docs/querying/api.md

```diff
@@ -936,6 +936,27 @@ $ curl http://localhost:9090/api/v1/status/tsdb
 
 *New in v2.15*
 
+
+## Remote Read API
+
+This API provides data read functionality from prometheus. This interface expects [snappy](https://github.com/google/snappy) compression.
+The API definition is located [here](https://github.com/prometheus/prometheus/blob/master/prompb/remote.proto).
+
+Request are made to the following endpoint.
+```
+/api/v1/read
+```
+
+### Samples
+
+This will return a message that includes a list of raw samples.
+
+### Streamed Chunks
+
+These streamed chunks utilize an XOR algorithm inspired by the [Gorilla](http://www.vldb.org/pvldb/vol8/p1816-teller.pdf)
```

Comment:
This is a little much detail, the relevant point is they're the internal implementation of the chunks.